content/courses/mobile/solana-mobile-dapps-with-expo.md

```diff
@@ -71,14 +71,20 @@ speakers. The libraries are intuitive and the documentation is phenomenal.
 #### How to create an Expo app
 
 To get started with Expo, you first need the prerequisite setup described in the
-[Introduction to Solana Mobile lesson](/content/courses/mobile/intro-to-solana-mobile).
+[Introduction to Solana Mobile lesson](/content/courses/mobile/intro-to-solana-mobile.md).
 After that, you'll want to sign up for an
-[Expo Application Services (EAS) account](https://expo.dev/).
+[Expo Application Services (EAS) account](https://expo.dev/signup).
 
 Once you have an EAS account, you can install the EAS CLI and log in:
 
 ```bash
+# For npm users
 npm install --global eas-cli
+
+# For yarn users
+yarn global add eas-cli
```

Comment:
We can remove `yarn`. 

1. We don't want to require users to install extra software, or add unneeded content, and npm these days is just as good as yarn
2. `yarn` got weird recently - v4 asks uses to install vscode plugins to support odd compression types
3. `pnpm` seems to be taking over as an alternative package manager

Reply:
I've been an active yarn user, but yeah it has started getting worse, I will replace package install snippets with pnpm